frontend/src/components/IssueList/FilteredIssueList.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import IssueList from './IssueList';
+
+export default (props) => {
+  const {data, filter, title} = props;
```

Comment:
is this the same filter? as data.filter ? I find that a bit confusing. Maybe issueFilter or something would prove slightly more useful?

Reply:
I would prefer `status`

Reply:
destructuring, like `const filter = props.filter;` but more compact than three separate statements/lines.

`status` is the db field, here we are passing a value to filter that field. I suppose it's short for `statusFilter` or even `issueStatusFilter`. These seem a bit lengthy?

I guess we could discuss this till we're blue in the face, so suggest @Mortuie reviews the 'evidence' and makes a ruling :)

Reply:
Nonono, I mean in `const { data, filter, title } = props;` why is it called filter... This is misleading as `data.filter` also exists and now we have a variable and a function with the same name. That's all I am saying happy with the rest of it!